src/redux/videoSlice.ts

```diff
@@ -489,6 +503,14 @@ const setThumbnailHelper = (state: video, id: Track["id"], uri: Track["thumbnail
   }
 };
 
+const ZOOM_SECONDS_VISIBLE = 20 * 1000;
```

Comment:
For videos of 20 seconds or less, I can not use the zoom-slider at all because the returned value is used as maximum and will always be 1. This might be fine since the zoom level is already very detailed on a short video, but it would probably make sense to have some kind of visual indication / explanation as to why the user can't drag the slider. I was a bit confused at first.

Reply:
Maybe even just hide the zoom bar completely for short videos < 20sec?

Reply:
Hiding is nice. On the other hand, we could always have a minimal "maximum zoom factor" of 2.

Reply:
Personally, I'd prefer to have a minimum zoom factor. That way the user always has the same experience and never has to wonder where the zoom slider has gone.

Reply:
I have changed the minimal maxZoom factor. This patch takes care of it: 
[0001-Update-the-minimal-maxZoom-from-1-to-2.patch.txt](https://github.com/user-attachments/files/15884514/0001-Update-the-minimal-maxZoom-from-1-to-2.patch.txt)

Reply:
@luniki Now the zoom bar seems to be not correctly shown for short videos: 

![image](https://github.com/opencast/opencast-editor/assets/32971521/0dbfa907-764a-4910-af7b-b12c7614ca2e)

Tested with "ID-3d-print" video.

Reply:
@luniki Now the zoom-bar seems to be not correctly shown for short videos. I guess the buttons are not intended? 

![image](https://github.com/opencast/opencast-editor/assets/32971521/4998bc1c-2a14-435e-b6ae-a959fd1859e8)

Tested with the `ID-3d-print` video on develop.

Reply:
@jduehring I haven't noticed your problems on https://test.editor.opencast.org/1033/2024-06-18_10-33-57/?id=ID-3d-print. My PC displays the zoom slider as it should (tested with Safari, Firefox, Chrome and Opera).

Reply:
Hmm strange, might have been a local hiccup on my side. Seems to work now. So all in all I have no more complaints and its ready to merge from my side.